### PR TITLE
Add Kerberos 5 support

### PR DESCRIPTION
**Pull request by julek-wolfssl**

### CODE REVIEW

**Review by haydenroche5 — CHANGES_REQUESTED** [review]

File: src/ssl.c [hunk 1]
```diff
@@ -60686,31 +60877,62 @@ int wolfSSL_i2d_PKCS7_bio(WOLFSSL_BIO *bio, PKCS7 *p7)
...
     }
 
     if ((len = wc_PKCS7_EncodeSignedData(p7, output, len)) < 0) {
         WOLFSSL_MSG("wc_PKCS7_EncodeSignedData error");
         goto cleanup;
     }
 
+    ret = len;
+cleanup:
+    if (p7->rng == &rng) {
```
[the remark this answers is not in the document]
Comment (haydenroche5) [reply src/ssl.c]: I don't see the `if` section you're talking about.

**Review by haydenroche5 — CHANGES_REQUESTED** [review]
Just one issue to resolve.

File: src/ssl.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (julek-wolfssl) [reply src/ssl.c]: Unless my git client is lying to me, this is part of the initialization of the function:
```
    if (!p7->rng) {
        if (wc_InitRng(&rng) != 0) {
            WOLFSSL_MSG("wc_InitRng error");
            return WOLFSSL_FAILURE;
        }
        p7->rng = &rng;
    }
```

[the remark this answers is not in the document]
Comment (haydenroche5) [reply src/ssl.c]: GitHub just isn't showing this snippet, but it's there.

**Review by haydenroche5 — APPROVED** [review]

**Review by dgarske — APPROVED** [review]

**Pull request merged**